test_scripts/API/AlertManeuver/015_ATF_Navi_IsReady_no_resp_AlertManeuver_Navi_AlertMan_UNSUP_RES.lua

```diff
@@ -0,0 +1,174 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS to at least one HMI-portions
+-- [Navigation Interface] SDL behavior in case HMI does not respond to Navi.IsReady_request
+--
+-- Description:
+-- test is intended to check that SDL sends WARNINGS (success:true) to mobile app in case HMI respond WARNINGS to at least one HMI-portions
+-- in this particular test it is checked case when TTS.Speak gets WARNINGS and Navi.AlertManeuver gets UNSUPPORTED_RESOURCE from HMI
+--
+-- 1. Used preconditions:
+-- HMI does not respond to Navi.IsReady
+-- App is registered and activated SUCCESSFULLY
+-- 2. Performed steps:
+-- MOB -> SDL: sends AlertManeuver
+-- HMI -> SDL: TTS.Speak (WARNINGS), Navi.AlertManeuver (UNSUPPORTED_RESOURCE)
+--
+-- Expected result:
+-- SDL -> HMI: resends TTS.Speak and Navi.AlertManeuver
+-- SDL -> MOB: AlertManeuver (result code: UNSUPPORTED_RESOURCE, success: true)
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+-- ToDo (vvvakulenko): remove after issue "ATF does not stop HB timers by closing session and connection" is resolved
+config.defaultProtocolVersion = 2
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require ('user_modules/shared_testcases/commonSteps')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForNavi_IsReady = require('user_modules/IsReady_Template/testCasesForNavi_IsReady')
+local mobile_session = require('mobile_session')
+config.SDLStoragePath = commonPreconditions:GetPathToSDL() .. "storage/"
+
+--[[ Local Variables ]]
+local storagePath = config.SDLStoragePath..config.application1.registerAppInterfaceParams.appID.. "_" .. config.deviceMAC.. "/"
+local ServerAddress = commonFunctions:read_parameter_from_smart_device_link_ini("ServerAddress")
+
+--[[ General Precondition before ATF start ]]
+commonFunctions:SDLForceStop()
+commonSteps:DeleteLogsFiles()
+commonSteps:DeletePolicyTable()
+testCasesForPolicyTable:precondition_updatePolicy_AllowFunctionInHmiLeves({"BACKGROUND", "FULL", "LIMITED"},"AlertManeuver")
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/connecttest_initHMI')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_InitHMI_OnReady()
+  testCasesForNavi_IsReady.InitHMI_onReady_without_Navigation_IsReady(self, 1)
```

Comment:
@vvvakulenko, the function that you are using is with different name in file:
testCasesForNavi_IsReady.lua. There it is named InitHMI_onReady_without_Navi_IsReady()